package.json

```diff
@@ -47,49 +47,56 @@
     "prelint": "cd samples; npm link ../; npm i"
   },
   "dependencies": {
-    "@google-cloud/projectify": "^1.0.0",
-    "@google-cloud/promisify": "^1.0.0",
-    "@grpc/grpc-js": "0.6.16",
+    "@google-cloud/projectify": "^1.0.4",
+    "@google-cloud/promisify": "^1.0.4",
+    "@grpc/grpc-js": "0.7.0",
```

Comment:
@alexander-fenster are we ready for that to sneak in?

Reply:
these dependencies are pre-existing, I just moves them around. They might for the samples-test, not sure about that, i can try remove them to see the outcome. Thank you!

Reply:
We need to use the same version as shipped with google-gax, otherwise it will cause confusion and double installs. Gax uses 0.6.x and will switch to 0.7.x a little bit later, so @xiaozhenliu-gg5 let's use 0.6.x here as well.

Reply:
Oh, that's what Justin means, move it back to `0.6.x` thanks!